### PR TITLE
test: refactored perp and price feed tests

### PR DESCRIPTION
**Pull request by onikonychev**
- Replaced Unittest with standard pytest approach https://docs.pytest.org/en/7.1.x/
- Extracted all "aux" methods into fixtures
- Removed duplicated code
- Added `conftest.pytest_configure()` which loads and validates the environment
- Refactored `perp` tests
- Refartored `pricefeed` tests

### CODE REVIEW

File: tests/perp_test.py [hunk 1]
```diff
@@ -1,47 +1,47 @@
...
-from nibiru import common
+from nibiru import Coin, common
+from tests import dict_keys_must_match, transaction_must_succeed
 
 PRECISION = 6
 
 
-class TestPerp(tests.ModuleTest):
-    def test_open_close_position(self):
+class TestPerp:
```
Comment (matthiasmatt) [inline tests/perp_test.py:12]: Do we still need a class if we don't use unittest anymore?

File: tests/pricefeed_test.py [hunk 2]
```diff
@@ -1,81 +1,74 @@
 # pricefeed_test.py
 from datetime import datetime, timedelta
 
-import tests
-from nibiru import common
+from tests import dict_keys_must_match, transaction_must_succeed
 
-from . import CONFIG
 
+class TestPricefeed:
```
Comment (matthiasmatt) [inline tests/pricefeed_test.py:7]: Same, we could remove this class

**Review by matthiasmatt — CHANGES_REQUESTED** [review]

File: tests/pricefeed_test.py (hunk 2 above)
Comment (onikonychev) [reply tests/pricefeed_test.py]: Indeed, this is just an extra indent now.

File: tests/perp_test.py (hunk 1 above)
Comment (onikonychev) [reply tests/perp_test.py]: Removed.

**Review by Unique-Divine — APPROVED** [review]

**Pull request merged**